themes/src/main/resources/theme/base/login/resources/js/passwordEntropy.js

```diff
@@ -0,0 +1,50 @@
+const clamp = (number, min, max) => {
+    return Math.max(min, Math.min(number, max));
+}
```

Comment:
Can be shorted by using the implicit return.
```suggestion
const clamp = (number, min, max) => Math.max(min, Math.min(number, max));
```